.gitignore

```diff
@@ -37,4 +37,4 @@ build/
 node_modules
 
 # Vagrant
-/tools/vagrant/.vagrant*
+.vagrant/
```

Comment:
`*` accounts for multiple vagrant files. can you add it back? (i have several for supporting various user scenarios)